docs/features/notifications.md

```diff
@@ -0,0 +1,131 @@
+# Notifications
+
+Argo Rollouts provides notifications powered by the [Notifications Engine](https://github.com/argoproj/notifications-engine).
+Controller administrators can leverage flexible systems of triggers and templates to configure notifications requested
+by the end users. The end-users can subscribe to the configured triggers by adding an annotation to the Rollout objects.
+
+## Configuration
+
+The trigger defines the condition when the notification should be sent as well as the notification content template.
+Default Argo Rollouts comes with a list of built-in triggers that cover the most important events of Argo Rollout live-cycle.
+Both triggers and templates are configured in the `argo-rollouts-notification-configmap` ConfigMap. In order to get
+started quickly, you can use pre-configured notification templates defined in [notifications-install.yaml](../../manifests/notifications-install.yaml).
+
+If you are leveraging Kustomize it is recommended to include [notifications-install.yaml](../../manifests/notifications-install.yaml) as a remote
+resource into your `kustomization.yaml` file:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- https://github.com/argoproj/argo-rollouts/releases/latest/download/install.yaml
+- https://github.com/argoproj/argo-rollouts/releases/latest/download/notifications-install.yaml
+```
+
+After including the `argo-rollouts-notification-configmap` ConfigMap the administrator needs to configure integration
+with the required notifications service such as Slack or MS Teams. An example below demonstrates Slack integration:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argo-rollouts-notification-configmap
+data:
+  service.slack: |
+    token: $slack-token
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argo-rollouts-notification-secret
+stringData:
+  slack-token: <my-slack-token>
+```
+
+Learn more about supported services and configuration settings in services [documentation](../generated/notification-services/overview.md).
```

Comment:
This link is broken

Reply:
link is referencing file is generated directory. So these files are not committed to git.